nemo/src/io/format_builder.rs

```diff
@@ -250,9 +257,11 @@ pub(crate) trait FormatBuilder: Sized + Into<AnyImportExportBuilder> {
     fn supports_tag(tag: &str) -> bool {
         Self::Tag::from_str(tag).is_ok()
     }
-    fn override_resource_spec(&self, _direction: Direction) -> Option<ResourceSpec> {
+
+    fn override_resource_builder(&self, _direction: Direction) -> Option<ResourceBuilder> {
         None
     }
+
```

Comment:
if you're still passing around the `ResourceBuilder` at this stage, this should rather look like 
```rust
fn customize_resource_builder(&self, _direction: Direction, builder: ResourceBuilder) -> ResourceBuilder {
  builder
}
```